test/test_indexing.expected

```diff
@@ -23,6 +23,28 @@ def arange(length: int, device: torch.device, *, _launcher=_default_launcher):
     _launcher(_helion_arange, (triton.cdiv(length, _BLOCK_SIZE_0),), out, out.stride(0), length, _BLOCK_SIZE_0, num_warps=4, num_stages=3)
     return out
 
+--- assertExpectedJournal(TestIndexing.test_arange_block_size_multiple)
+from __future__ import annotations
+
+import torch
+import triton
+import triton.language as tl
+from helion.runtime import default_launcher as _default_launcher
+
+@triton.jit
+def _helion_arange_block_size_mul(out, _BLOCK_SIZE_0: tl.constexpr, mul_2: tl.constexpr):
```

Comment:
Lifting `mul_2` to become triton function input is key for making `tl.arange(0, mul_2)` work